vips/vip-152.ts

```diff
@@ -0,0 +1,114 @@
+    againstDescription:
+      "I do not think that Venus Protocol should proceed with setting the vBNB admin and setting up PSR",
+    abstainDescription:
+      "I am indifferent to whether Venus Protocol proceeds with setting the vBNB admin and setting up PSR",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: VBNBAdmin_ADDRESS,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: vBNB_ADDRESS,
+        signature: "_setPendingAdmin(address)",
+        params: [VBNBAdmin_ADDRESS],
+      },
+      {
+        target: VBNBAdmin_ADDRESS,
+        signature: "_acceptAdmin()",
+        params: [],
+      },
+      {
+        target: PSR,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: VBNBAdmin_ADDRESS,
+        signature: "setProtocolShareReserve(address)",
+        params: [PSR],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrUpdateDistributionConfigs(DistributionConfig[])", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrUpdateDistributionConfigs(DistributionConfig[])", FAST_TRACK_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrUpdateDistributionConfigs(DistributionConfig[])", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "removeDistributionConfig(Schema,address)", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "removeDistributionConfig(Schema,address)", FAST_TRACK_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "removeDistributionConfig(Schema,address)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrRemoveAssetFromPrime(address,bool)", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrRemoveAssetFromPrime(address,bool)", FAST_TRACK_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrRemoveAssetFromPrime(address,bool)", NORMAL_TIMELOCK],
+      },
+      {
+        target: PSR,
+        signature: "addOrUpdateDistributionConfigs((uint8,uint8,address)[])",
+        params: [
+          [
+            [0, 50, RISK_FUND],
+            [0, 50, TREASURY],
+            [1, 40, RISK_FUND],
+            [1, 60, TREASURY],
+          ],
+        ],
```

Comment:
I would also set the pool registry in the VIP:
* target: PSR
* signature: setPoolRegistry(address)
* params: ["0x9F7b01A536aFA00EF10310A162877fd792cD0666"]

I think this way we'll avoid unexpected behaviors in the future